app/controllers/terms_controller.rb

```diff
@@ -22,8 +22,8 @@ def create
     @vocabulary = find_vocabulary
     combined_id = CombinedId.new(params[:vocabulary_id], term_params[:id])
     term_form = term_form_repository.new(combined_id)
-    term_form.attributes = term_params.except(:id)
-    term_form.set_languages(params)
+    term_form.attributes = vocab_params.except(:id)
```

Comment:
I definitely think its a good win to look into how to adjust the way the data is being passed through the form to make it more straight forward, Ill create a ticket and tag it with enhancement and we can definitely look into it.